Makefile

```diff
@@ -17,5 +18,10 @@ test:
 
 .PHONY: lint
 lint:
-	hlint .
-	weeder .
+	# hlint
+	curl -sL https://raw.github.com/ndmitchell/hlint/master/misc/travis.sh | sh -s -- --version
+	curl -sL https://raw.github.com/ndmitchell/hlint/master/misc/travis.sh | sh -s .
```

Comment:
Can you describe the benefits of using this custom script over the simpler:

```
stack install --copy-compiler-tool hlint weeder
stack exec hlint
stack exec weeder
```

Reply:
This script is able to use always latest version hlint (and weeder).

```
stack install --copy-compiler-tool hlint weeder
```

By above instruction is installed binaries of different version depends on that resolvers (`STACK_YAML` env).

Reply:
Gotcha. I guess I could go either way: always latest sounds good, but using a tool appropriate for the compiler/dependencies versions under test could also be required in some edge-cases.

You've got it Green, so we can just go with this for now!

Reply:
Good point! Revert changes?

Reply:
No, I think it's OK. We'll see how it goes.

Reply:
OK! :)